package.json

```diff
@@ -8,7 +8,7 @@
   "private": true,
   "devDependencies": {
     "eslint-config-prettier": "^2.9.0",
-    "pocket-build-tools": "git+https://git@github.com/Pocket/front-end-build-tools.git",
+    "pocket-build-tools": "git+https://git@github.com/Pocket/front-end-build-tools.git#add/circleCI",
```

Comment:
This will be reverted after merge of https://github.com/Pocket/front-end-build-tools/pull/13